doc/source/security/data-masking.rst

```diff
@@ -18,7 +16,7 @@ The following command installs the plugin:
 
 .. code-block:: bash
 
-    INSTALL PLUGIN data_masking SONAME 'data_masking.so';
+    shell>INSTALL PLUGIN data_masking SONAME 'data_masking.so';
 
```

Comment:
should we use mysql as code block interpreter for syntax highlighting

Reply:
Fixed.